### DISCUSSION TITLE
Checklists ?

### DISCUSSION THREAD

**Discussion by linuxguy123** [nasa/openmct#3875] [Community Q&A]
Anyone else have checklists as part of their mission activities ?

Thoughts on how they could be integrated into OpenMCT ?

**Comment by jvigliotta** [comment]
I'm not aware of anything like that currently, but I could see that be a pretty useful little plugin. It's not in any future development plans for Open MCT but if someone has built something like that, we'd love to see it in the [Show & Tell](https://github.com/nasa/openmct/discussions/categories/show-and-tell) section of our discussions.
